compose/service.py

```diff
@@ -798,6 +801,10 @@ def _get_container_host_config(self, override_options, one_off=False):
             init_path = options.get('init')
             options['init'] = True
 
+        nano_cpus = None
+        if 'cpus' in options:
+            nano_cpus = int(options.get('cpus') * 1000000000)
```

Comment:
We should check in `compose/config/validation.py` that the result of `cpus * 10^9` is not a decimal value, [same as here](https://github.com/moby/moby/blob/v17.05.0-ce/opts/opts.go#L378-L381)